app/helpers/application_helper/button/server_level_options.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicationHelper::Button::ServerLevelOptions < ApplicationHelper::Button::Basic
+  needs :@record, :@sb
+
+  def visible?
+    x_active_tree == :diagnostics_tree &&
```

Comment:
@jzigmund Sure, I can, but I'm thinking what it would change.

The `x_active_tree` is just wrapper around `@sb`:
```ruby
# app/controllers/mixins/sandbox.rb:

 31   def x_active_tree                                                            
 32     sandbox[:active_tree]                                                      
 33   end
```

And because in the class definition of this button is the `@sb` variable already mentioned as required, the button will be skipped, if the `@sb` is `nil`. Changing 

```diff
- x_active_tree
+ @sb[:active_tree]
```

would make no difference.

And if the `x_node_tree` should be replaced, I'd expect the same to be done with `x_node` *(also wrapper for `@sb`)*:

```ruby
# app/controllers/mixins/sandbox.rb:

188   def x_node(tree = nil)                                                       
189     tree ||= x_active_tree                                                     
190     sandbox.fetch_path(:trees, tree, :active_node)                             
191   end
```